test/bidi.spec.mjs

```diff
@@ -133,24 +133,24 @@ describe('bidi.js', function() {
             });
         });
 
-        describe('thai contexts', () => {
+        describe('thai contexts', function () {
             it('should match thai words in a given text', () => {
                 const tokenizer = bidiThai.tokenizer;
-                tokenizer.tokenize('The king said: เป็นคนใจดีสำหรับทุกคน because ความรักคือทุกสิ่ง');
+                bidiThai.processText('The king said: เป็นคนใจดีสำหรับทุกคน because ความรักคือทุกสิ่ง');
```

Comment:
I'm not sure what that does, can you explain please? Do we need to mix it into this PR?

Reply:
`tokenizer.tokenize` did not registerContextChecker("thaiWord") so the test failed
`processText` however call tokenizeText() which register it here is the stacktrace

```
  1) bidi.mjs
       thai scripts
         thai contexts
           should match thai words in a given text:
     TypeError: ranges.map is not a function
      at Context.<anonymous> (file:///home/yne/opentype.js/test/bidi.spec.mjs:141:38)
      at process.processImmediate (node:internal/timers:478:21)

```